aspnetcore/blazor/hosting-models.md

```diff
@@ -18,9 +18,34 @@ NOTE: Daggered lines under the table (&dagger;, &Dagger;) use a double-space at
 
 -->
 
-This article explains the different Blazor hosting models and how to choose which one to use.
+:::moniker range=">= aspnetcore-8.0"
 
-:::moniker range=">= aspnetcore-6.0"
+This article explains Blazor hosting models, which can be applied in different parts of a Blazor app at either compile time or runtime.
+
+:::moniker-end
+
+:::moniker range="< aspnetcore-8.0"
+
+This article explains Blazor hosting models and how to choose which one to use.
+
+:::moniker-end
+
+:::moniker range=">= aspnetcore-8.0"
+
+Blazor is a web framework for building web UI components ([Razor components](xref:blazor/components/index)) that can be hosted in different ways:
+
+* Server-side in ASP.NET Core (*Blazor Server* or statically rendered).
+* Client-side in the browser on a [WebAssembly](https://webassembly.org/)-based .NET runtime (*Blazor WebAssembly*).
+* Blazor Hybrid to build native client apps.
+
+> [!NOTE]
+> For web-based Blazor apps, a component's hosting model is set by its *render mode*, which is described with examples in <xref:blazor/fundamentals/render-modes>. We recommend waiting to read the *Render modes* article until after you've read the articles leading up to it, as it relies on an understanding of broader Blazor concepts.
```

Comment:
> We recommend waiting to read the *Render modes* article until after you've read the articles leading up to it, as it relies on an understanding of broader Blazor concepts.

This seems pretty awkward to have to repeat everywhere. I think we should just remove this advisory. Most of our doc articles require some sort of prior knowledge to fully understand them.